addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -166,18 +166,25 @@ function factory(dependencies) {
                 }
             }
 
-            if (previous.activityIds.join(',') !== this.activityIds.join(',')) {
+            if (!this.thread || this.thread.isTemporary) {
+                return;
+            }
+            if (
+                !previous.activityIds ||
+                previous.activityIds.join(',') !== this.activityIds.join(',')
+            ) {
```

Comment:
Yeah, it does... but the fix is not complete because it still shows activity from previous thread. It just avoid opening the dialog as a consequence of refresh activities.

I think it should refresh activities, but this consists of no activities on chatter (because temporary thread).

Reply:
done